src/lib.rs

```diff
@@ -141,14 +141,14 @@
 //! * Windows platforms targeting MinGW (e.g. your target triple ends in `-gnu`)
 //!   require `cc` to be available in `PATH`. We recommend the
 //!   [MinGW-w64](https://www.mingw-w64.org/) distribution, which is using the
-//!   [Win-builds](http://win-builds.org/) installation system.
+//!   [Win-builds](https://github.com/alex85k/winbuilds/blob/master/README.md) installation system.
```

Comment:
Do we really need to link to win-builds? Why do users need to know what mingw-w64 is built on?

I'm asking because I think we should be very careful with third party links from official rust-lang documentation.

Reply:
I think maybe we can just say we recommend `MinGW-w64` and leave it at that.

Reply:
Thanks, I've removed it